neo/Persistence/Memory/Store.cs

```diff
@@ -0,0 +1,70 @@
+using Neo.IO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace Neo.Persistence.Memory
+{
+    public class Store : IStore
```

Comment:
I think we should use different names for different implementations. Like LevelDBStore / MemoryStore